jsonapi/fixtures_test.go

```diff
@@ -361,86 +449,110 @@ func (q Question) GetReferencedStructs() []MarshalIdentifier {
 
 type Identity struct {
 	ID     int64    `json:"-"`
+	LID    int64    `json:"-"`
 	Scopes []string `json:"scopes"`
 }
 
-func (i Identity) GetID() string {
-	return fmt.Sprintf("%d", i.ID)
+func (i Identity) GetID() Identifier {
+	id := Identifier{ID: fmt.Sprintf("%d", i.ID), Name: "identities"}
```

Comment:
No need to define Name, since GetName was not implemented before

Reply:
removed